svg/historical.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<title>Historical SVG features must be removed</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+var removedInterfaces = [
+  "SVGAnimatedPathData",
+  "SVGColor",
+  "SVGDocument",
+  "SVGElementInstanceList",
+  "SVGExternalResourcesRequired",
+  "SVGFilterPrimitiveStandardAttributes",
+  "SVGICCColor",
+  "SVGLangSpace",
+  "SVGPaint",
+  "SVGPathSeg",
+  "SVGStylable",
```

Comment:
Ah, "Turned SVGLocatable and SVGTransformable into SVGGraphicsElement, which directly inherits from SVGElement" not found when searching for "remove" :) I can't find in SVGTransformable https://www.w3.org/TR/SVG/single-page.html so I guess I'll omit that.